doc/source/dev/contributor/adding_new/new_stats_distribution.rst.inc

```diff
@@ -0,0 +1,93 @@
+Adding A New Statistics Distribution
+------------------------------------
+
+For hundreds of years statisticians, mathematicians and scientists have needed
+to understand, analyze and model data.
+This has led to a plethora of statisics distributions, many of which are
+related to others.
+Modeling of new types of data continues to give rise to new distributions,
+as does theoretical considerations being applied to new disciplines.
+SciPy models about a dozen discrete distributions
+:ref:`discrete-random-variables` and 100 continuous distributions
+:ref:`continuous-random-variables`.
+
+To add a new distribution, a good reference is needed.
+Scipy typically uses [JKB]_ as its gold standard, with WikipediaDistributions_
+articles often providing some extra details and/or graphical plots.
+
+How to create a new continuous distribution
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+There are a few steps to be done to add continuous distribution XXXXX to SciPy.
+(Adding a discrete distribution is similar).
+
+
+Before Implementation
+.....................
+
+1. See if it already been implemented (that saves a lot of effort!)
+
+  - It may have been implemented with a different name.
+  - It may have been implemented with a different parameterization (shape parameters).
+  - It may be a specialization of a more general family of distributions.
+
+It is very common for multiple disciplines to discover/rediscover a
+distribution (or a specialization or different parameterization).
+There are a few existing SciPy distributions which are specializations
+of other distributions.  E.g. The :class:`scipy.stats.arcsine` distribution
+is a specialization of the :class:`scipy.stats.beta` distribution.
+These duplications exist for (very!) historical and widespread usage reasons.
+At this time, adding new specializations/reparametrizations of existing
+distributions to SciPy is not supported,  mainly due to the increase in user
+confusion resulting from such additions.
+
+2. Create a SciPy Issue on github listing the distribution, references and
+   reasons for its inclusion.
+
+
+Implementation
+..............
+
```

Comment:
Here it would be great if you can just use a valid word instead of `XXXXX` or at least mention that you have used it as a placeholder, say, _Suppose you want to add XXXXX distribution to Scipy..._ 

I would still use a valid word, _squirrel_, _icecream_, _vanrossum_ etc. to make it more readable.

Reply:
Ah nevermind, didn't see the introduction mentioned this. Sorry.

Reply:
Actually I like "squirrel" better than XXXX!